### PR TITLE
doc : Add contribution guidelines

### PR DESCRIPTION
**Pull request by purohitdheeraj**
Issue No. #21 

**Summary:**

I have added contributing guidelines in `CONTRIBUTING.md`
some keys section added in this commit are
- how to contribute (fork, pull request)
- basic info on communication, issues, attribution 

**Changes:**
- `CONTRIBUTING.md` file

do let me know if we can make it more better, thank you

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# Contributing Guidelines
+
+Welcome to the jslovers project! We appreciate your interest in contributing. Before you get started, please take a moment to review these guidelines to ensure a smooth and productive collaboration.
```
Comment (Neha) [inline CONTRIBUTING.md:3]: Can you please make "jslovers" to "JSLovers"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+8. **Create a Pull Request**: Open a pull request from your fork to this main repository. Please provide a detailed description of your changes and reference any relevant issues if applicable.
+
+9. **Review and Discussion**: Your pull request will undergo review by the maintainers and other contributors. Be prepared to address feedback and make necessary changes.
+
+10. **Merge**: Once your pull request is approved, it will be merged into the main branch.
+
+## Coding Standards
+
+Please follow the coding standards and guidelines specific to the "jslovers" project. If you're unsure about any coding standards, you can check the project's `CONTRIBUTING.md` or ask for clarification in your pull request or by reaching out to maintainers.
```
Comment (Neha) [inline CONTRIBUTING.md:55]: Please change 'jslovers' to "JSLovers"

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+You can contribute not only through code but also by helping to identify and resolve issues. Feel free to browse our issue tracker, pick an issue that interests you, and leave a comment that you're working on it.
+
+## Communication
+
+If you have questions or need assistance, you can reach out to the project maintainers through GitHub issues or other communication channels, such as a dedicated Discord server or Slack workspace (if applicable).
+
+## Attribution
+
+Contributors will be credited for their contributions in the project's README or a dedicated contributors file.
```
Comment (Neha) [inline CONTRIBUTING.md:67]: We don't have any contribution files. Nor, we are going to add in the Project's readme as of now. Please as of now, remove this line.

The contribution will be visible on the GitHub.

**Review by Neha — COMMENTED** [review]
All looks good. There are a few small changes. Please do it & then we are ready for merging

**Comment by purohitdheeraj** [comment]
Hey @Neha thanks for the review
i did the required changes in new commit

**Comment by Neha** [comment]
Awesome!! thank you for contribution

**Pull request merged**